README.md

```diff
@@ -131,6 +131,23 @@ Or enforce by running [scripts/test.sh](scripts/test.sh) as part of your ci/cd p
 
 ![FWI calculator container diagram](container_diagram.png)
 
+## Config
```

Comment:
Let's rather just be specific in mentioning the existence of the global.config.yaml and ????secret???.yaml file, and have the descriptions in the yaml files themseles. We could add here an example of how to apply the template, e.g:

```
oc -n openshift-project process -f openshift/templates/global.config.yaml -p WFWX-AUTH-URL=https://spotwx.com/
```